specification/sql/resource-manager/Microsoft.Sql/preview/2021-05-01-preview/Databases.json

```diff
@@ -787,6 +787,44 @@
         }
       }
     },
+    "DatabaseIdentity": {
```

Comment:
Could you leverage https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v3/managedidentity.json instead?
It greatly simplifies reviewing for consistency.

Reply:
we might not carry https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v3/managedidentity.json this time, due to it is just added 16 days ago. 
however, all our swagger files in this PR is auto-generated by SQL repo. Change the reference would need to go through all the test process again. 
We will address this issue in the separate PR.